src/Uno.PackageDiff/Writers/XmlIngnoreWriter.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using Mono.Cecil;
+using NuGet.Versioning;
+
+namespace Uno.PackageDiff.Writers
+{
+	internal class XmlIngnoreWriter : IDiffWriter
```

Comment:
```suggestion
	internal class XmlIgnoreWriter : IDiffWriter
```
and, the same in commit+pr title too